addons/mail/static/src/components/notification_alert/notification_alert.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <t t-name="mail.NotificationAlert" owl="1">
-        <div t-if="messaging and messaging.isNotificationBlocked" class="o_NotificationAlert alert alert-primary d-flex justify-content-center" t-attf-class="{{ className }}" t-ref="root">
+    <t t-name="mail.NotificationAlertView" owl="1">
+        <div t-if="messaging and messaging.isNotificationBlocked" class="o_NotificationAlertView alert alert-primary d-flex justify-content-center" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
It looks wrong to have a top level condition like this. Creation of view record should be dependent on these conditions and not exist if it makes no sense.